container/runner_py38_mantid.Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM continuumio/miniconda3 AS build
+
+ARG MANTID_VERSION
+WORKDIR /app
+ADD . .
+
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update &&\
+    apt-get install -y \
+    wget \
+    gnupg \
+    git \
+    software-properties-common \
+    python3-dev \
+    default-libmysqlclient-dev \
+    build-essential \
+    gcc
+
+# Install conda-pack into the base environment
+RUN conda install conda-pack
```

Comment:
Is `conda-pack` necessary as you can install an environment to a specific directory using `--prefix`?